fusion-cli/entries/client-public-path.js

```diff
@@ -4,7 +4,8 @@ declare var __webpack_public_path__: string;
 declare var __webpack_nonce__: string;
 */
 
-import '@babel/polyfill';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```

Comment:
If we're using standalone regenerator, let's use the non-import side effect option, then we can remove this from our side effect import whitelist

Reply:
Oh wait, nevermind, they removed that in favor of import side effects again